packages/eslint-config-godaddy-react/index.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   extends: [
     require('./extends')('eslint-config-godaddy'),
     'plugin:eslint-plugin-react/recommended'
+    'eslint-plugin-jsx-a11y'
```

Comment:
Shouldn't this be `'plugin:jsx-a11y/recommended'` here for extend?

Reply:
I guess? Not sure how to test given that this is the test command for this subpackage:

```
"test": "echo ok",
```